configurations/kms-ear.yaml

```diff
@@ -0,0 +1,11 @@
+
+scylla_encryption_options: "{ 'cipher_algorithm' : 'AES/ECB/PKCS5Padding', 'secret_key_strength' : 128, 'key_provider': 'KmsKeyProviderFactory', 'kms_host': 'kms_test'}"
+
+# config kms_hosts for AWS KMS service account
+append_scylla_yaml: |
+  kms_hosts:
+      kms_test:
+          master_key: 'alias/kms_encryption_test'
+          aws_region: 'us-east-1'
```

Comment:
The job ran in the `eu-west-1` region but here we have different one -> `us-east-1`.
How then it works? Is it ignored for now?

Reply:
it's actually doesn't matter the key we are using is needs to be in this region.

KMS code doesn't know what's the current region, and works only by this configuration.
we can raise an issue for it to be able to figure out the region if not specified